### PR TITLE
Additional systematic variations

### PR DESCRIPTION
**Pull request by sroychow**
Adding Prefire and alphaS variations. 
Fixing variable and weight definitions to handle both central and private MC nano inputs via the same code.
A utility file is also added in the Fit directory to handle systematics in the Fit.

### CODE REVIEW

File: analysisOnData/runAnalysisOnWJetsMC.py [hunk 1]
```diff
@@ -91,7 +90,7 @@ def RDFprocessWJetsMC(fvec, outputDir, sample, xsec, fileSF, ncores, pretendJob,
                 p.branch(nodeToStart = 'defs', nodeToEnd = 'defsAC', modules = steps)
                 p.branch(nodeToStart = 'defsAC', nodeToEnd = '{}/templatesAC_{}/{}Vars'.format('WToMu', region, s), modules = [ROOT.templateBuilder(wtomu_cut, var_weight,vars_vec,variations[1], 3)])
                 #reco templates for out of acceptance events
-                wtomu_cut+= "&& GenV_preFSR_qt>32. && GenV_preFSR_yabs>2.4"
```
Comment (emanca) [inline analysisOnData/runAnalysisOnWJetsMC.py:94]: @sroychow

Comment (emanca) [reply analysisOnData/runAnalysisOnWJetsMC.py:94]: this line puzzles me

Comment (sroychow) [reply analysisOnData/runAnalysisOnWJetsMC.py:94]: @emanca This was in the commented out section in the Wjets config[[here](https://github.com/emanca/wproperties-analysis/blob/master/analysisOnData/runAnalysisOnWJetsMC.py#L94)]. So I changed it to be consistent with the branch names in the private MC.

File: analysisOnData/data/systematics.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 systematics = {
     #"puWeight"  : (["puWeightUp", "puWeightDown"],"puWeightVars"),
+    "PrefireWeight"  : (["PrefireWeight_Up", "PrefireWeight_Down"],"PrefireWeightVars"),
```
Comment (emanca) [inline analysisOnData/data/systematics.py]: can you change PrefireWeight_Up -> PrefireWeightUp and down also?

Comment (sroychow) [reply analysisOnData/data/systematics.py]: yes can be done

**Review by emanca — APPROVED** [review]

**Pull request merged**